packages/components/src/components/form/SegmentSlider/SegmentSlider.tsx

```diff
@@ -0,0 +1,242 @@
+import styled, { css } from 'styled-components';
+
+import { borders, spacingsPx, typography } from '@trezor/theme';
+
+const DEFAULT_SEGMENT = { max: 0, name: '' };
+
+const thumb = css<{ disabled?: boolean }>`
+    appearance: none;
+    background: white;
+    border-radius: ${borders.radii.full};
+    box-shadow: 0 0 4px 0 rgb(0 0 0 / 50%);
+    margin-top: calc((${spacingsPx.xxs} - ${spacingsPx.xl}) / 2);
```

Comment:
If it has to be 10px, just use that value, no need to calculate it from the spacings.